pkg/codegen/go/gen_crd2pulumi.go

```diff
@@ -0,0 +1,69 @@
+package gen
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"path"
+
+	"github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+)
+
+// GenCRDTypes returns a map from each module name to a buffer containing the
+// code for its generated types.
+func GenCRDTypes(tool string, pkg *schema.Package) (map[string]*bytes.Buffer, error) {
```

Comment:
```suggestion
// CRDTypes returns a map from each module name to a buffer containing the
// code for its generated types.
func CRDTypes(tool string, pkg *schema.Package) (map[string]*bytes.Buffer, error) {
```

Linter is complaining about the name